examples/analyse_output.py

```diff
@@ -5,7 +5,7 @@
 
 if not os.path.isfile("./example.nc"):
     geo = Geometry(Nx=100, Ny=50, Lx=10, Ly=10, dt=0.1, T=20,)
-    bm = Model(blob_shape="exp", num_blobs=1000)
+    bm = Model(geometry=geo, blob_shape="exp", num_blobs=1000)
```

Comment:
Actually I didn't notice the geometry was not passed. What I meant is that you can rearrange the constructors in such a way that these examples still work in the same way as before (without having making the user initialize the geometry by himself each time). I don't really mind the current impl, but I thought you wanted to keep the same simple interface for the "base model case".

Reply:
well one doesn't have to specify the geometry sicne there Geometry __init__ has default values, I guess one can provide an example without a defined geometry and keep it where it is needed